tests-e2e/util/VideoPage.ts

```diff
@@ -0,0 +1,20 @@
+import { Locator, Page, Response } from "@playwright/test"
+import { CoursePage } from "."
+
+export class VideoPage {
+  readonly page: Page
+  readonly coursePage: CoursePage
+  readonly tabs: Locator
+  readonly tabPanels: Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.coursePage = new CoursePage(page, "course")
```

Comment:
**Suggestion** IMO, there's enough overlap between a "Video Page" and a "Video Embed", that it makes sense to have a single model like "Video Section" _which I realize isn't a great name._ Then in a test you'd use both:
```ts
  const course = new CoursePage(page, "course")
  await course.goto("/resources/ocw_test_course_mit8_01f16_l01v01_360p")
  const video = new VideoSection(page)
```

**Requested Change:** I do think it is worthwhile to...
1. Scope the tab selectors to the video area. *scoping is good, and It's not out of the question that we add "tab"/"tabpanel" to the course info tab.*
2. Make `tabs` and `tabpanels` methods that optionally accept options. *I also think singular naming is more in-line with how playwright does things

My recommendation would be:
```ts 

type ByRoleOptions = Parameters<Page["getByRole"]>[1]
class VideoSection {
  /**
   * Scope locators within the video area.
   */
  readonly within: Locator

  constructor(page: Page) {
    this.within = page.locator(".video-page")
  }


  tab(opts: ByRoleOptions): Locator {
    return this.within.getByRole("tab", opts)
  }

  tabpanel(opts: ByRoleOptions): Locator {
    return this.within.getByRole("tabpanel", opts)
  }
}
```